nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -141,6 +138,25 @@ ignore = [
 
     # Produces a lot of issues in the current codebase.
     "D401",  # First line of docstring should be in imperative mood
+    "D407",  # Missing dashed underline after section
+    "D416",  # Section name ends in colon
+]
+
+[tool.ruff.lint.pydocstyle]
+convention = "google"
+
+[tool.ruff.per-file-ignores]
+"{{ cookiecutter.app_name }}/migrations/*" = [
+    "D1",  # pydocstyle
```

Comment:
Does just "D" work?

Reply:
I'm not sure with this. There is another set of rules prefixed with "DTZ" (flake8-datetimez), and I can't find, whether just "D" includes "DTZ" or not, in the official documentation. I would rather keep it as is.

Reply:
As we move over to ruff more and more, my preference would be to find out for sure. I won't hold up this PR, just letting my preference be known.

Reply:
Per [ruff docs](https://docs.astral.sh/ruff/configuration/), this seems like it is fine

```toml
[lint]
# Enable Pyflakes (`F`) and a subset of the pycodestyle (`E`)  codes by default.
# Unlike Flake8, Ruff doesn't enable pycodestyle warnings (`W`) or
# McCabe complexity (`C901`) by default.
select = ["E4", "E7", "E9", "F"]
ignore = []
```

It states to enable Pyflakes, not to enable ones that start with F.

Reply:
Replaced with "D" only.